parl/agent_zoo/agents.py

```diff
@@ -0,0 +1,74 @@
+#   Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
```

Comment:
Change the file path to: parl/agents/simple_rl_agent.py

Reply:
done